src/styles/components/_component-grid.scss

```diff
@@ -37,16 +37,20 @@ $breakpoint-accordion-large-755: 47rem;
       @include doodle;
 
       width: 50%;
-      height: 5rem;
+      height: 100%;
       background-image: url("../../../public/images/doodles/doodle_line.png");
       background-size: 80%;
       rotate: -10deg;
       top: 0.5rem;
-      right: 4rem;
+      right: -1rem;
       background-position: 0;
+
+      @media only screen and (min-width: $breakpoint-accordion-medium-550) {
+        height: 5rem;
+      }
```

Comment:
Oh heck yeah, you solved the disappearing accordion bug! I was totally befuddled by this when I replaced the accordion image last night 🙂